src/test/java/com/ryanharter/auto/value/parcel/AutoValueParcelExtensionTest.java

```diff
@@ -250,9 +250,6 @@
         + "@AutoValue public abstract class Test implements Parcelable {\n"
             + "public abstract List<Parcelable1> a();\n"
             + "public abstract int[] b();\n"
-        + "public int describeContents() {\n"
-        + "return 0;\n"
-        + "}\n"
```

Comment:
Had to be removed for the test to pass because it was causing the method to not be emitted! Win.